.changeset/afraid-cooks-shave.md

```diff
@@ -0,0 +1,6 @@
+---
+'@sap-ai-sdk/orchestration': minor
+---
+
+[Compatibility Note] Deprecate `buildAzureContentFilter()` function.
+Use `buildAzureContentSafetyFilter()` function instead and use constants `ALLOW_SAFE`, `ALLOW_SAFE_LOW`, `ALLOW_SAFE_LOW_MEDIUM` or `ALLOW_ALL` to configure the severity levels of each Azure filter category.
```

Comment:
The values are not constants, they are string literal type. I would exclude it, or just mention severity levels without mentioning technical details

Reply:
But, isn't it good if we mention in the release notes that people cannot pass numbers anymore?

Shall I say:

Use `buildAzureContentSafetyFilter()` function instead and use descriptive string literals to configure the severity levels of each Azure filter category.